setup.py

```diff
@@ -1,9 +1,13 @@
 #!/usr/bin/env python
 from setuptools import setup, find_packages
 
+with open('viper/common/version.py') as f:
```

Comment:
Why not just importing the version (see: https://github.com/MISP/PyMISP/blob/master/setup.py#L9) ?